client/platform/desktop/backend/server.ts

```diff
@@ -100,18 +139,33 @@ router.register('/api/media', withErrorHandler((req, res) => {
 
   const range = ranges[0];
   // eslint-disable-next-line no-param-reassign
-  res.statusCode = 206;
+  res.status(206);
   res.setHeader('Content-Length', range.end - range.start + 1);
   res.setHeader('Content-Range', `bytes ${range.start}-${range.end}/${filestat.size}`);
-  if (req.method === 'HEAD') return res.end();
+  if (req.method === 'HEAD') {
+    return res.end();
+  }
   pump(fs.createReadStream(path, range), res);
   return null;
-}));
-
-// We need a server which relies on our router
-const server = http.createServer((req, res) => {
-  const handler = router.route(req);
-  handler.process(req, res);
 });
 
-export default server;
+app.use(cors({ origin: 'http://localhost:8080' }));
```

Comment:
very minor and only applies to `yarn serve:electron` - this locks you into CORS only being accepted from localhost:8080, so if you have other things running on that port it will fail to give to access because vue-cli-server will switch to 8081 or others.  Like I said minor, but just something to note.

Reply:
No, this is important.  It would be better to find a way to discover the devserver address and pull from that.